logging/unit_tests/test__gax.py

```diff
@@ -97,33 +115,47 @@ def _list_entries_with_paging_helper(self, payload, struct_pb):
 
         from google.api.monitored_resource_pb2 import MonitoredResource
         from google.logging.v2.log_entry_pb2 import LogEntry
-        from google.cloud._testing import _GAXPageIterator
         from google.cloud._helpers import _datetime_to_pb_timestamp
+        from google.cloud._helpers import UTC
+        from google.cloud._testing import _GAXPageIterator
+        from google.cloud.logging.client import Client
+        from google.cloud.logging.entries import StructEntry
+        from google.cloud.logging.logger import Logger
 
         SIZE = 23
         TOKEN = 'TOKEN'
         NEW_TOKEN = 'NEW_TOKEN'
         resource_pb = MonitoredResource(type='global')
-        timestamp_pb = _datetime_to_pb_timestamp(
-            datetime.datetime.utcnow())
-        entry_pb = LogEntry(log_name=self.LOG_NAME,
+        timestamp = datetime.datetime.utcnow().replace(tzinfo=UTC)
+        timestamp_pb = _datetime_to_pb_timestamp(timestamp)
+        entry_pb = LogEntry(log_name=self.LOG_PATH,
```

Comment:
`log_name=self.LOG_PATH`?
I know this is from [here](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2636/files#diff-41b5ab9e98931dbeb0d6ea726a819a1eR42), but it just reads in an unclear way.

Seeing how these are used later on, I'm not really sure I have any good suggestions on how to make this clearer.